client/src/components/my-dropdown.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Dropdown } from 'react-bootstrap';
+import * as uuid from 'uuid';
+import PropTypes from 'prop-types';
+
+class MyDropdown extends React.Component {
+  render() {
+    const { id, name, items } = this.props;
+    return (
+      <Dropdown className="mr-3">
+        <Dropdown.Toggle id={id} className="m-button">{name}</Dropdown.Toggle>
+        <Dropdown.Menu>
+          {items.map((item) => (
+            <Dropdown.Item
+              className="m-dropdown-item"
+              key={uuid.v4()}
```

Comment:
Why do you use this additional library, but not item.id?

Reply:
If it is array with string and not objects with property id, why don't you use array index? As I understand you don't change this array, so nothing bad will happen

> Let me explain, a key is the only thing React uses to identify DOM elements. What happens if you push an item to the list or remove something in the middle? If the key is same as before React assumes that the DOM element represents the same component as before. But that is no longer true